### ISSUE TITLE
assignment1-5

### ISSUE THREAD

**Issue by JinwoongKim** [JinwoongKim/cs231n#5]

**Comment by JinwoongKim** [comment]
```python
for lr in learning_rates:
    for reg in regularization_strengths:
        svm = LinearSVM()
        svm.train(X_train_feats, y_train, learning_rate=lr, reg=reg, num_iters=1500, verbose=False)
        y_train_pred = svm.predict(X_train_feats)
        y_val_pred = svm.predict(X_val_feats)
        train_accuracy = np.mean(y_train == y_train_pred)
        val_accuracy = np.mean(y_val == y_val_pred)
        print('train accuracy : ',train_accuracy)
        print('val accuracy : ', val_accuracy)
        results[(lr,reg)] = (train_accuracy, val_accuracy)
        if val_accuracy > best_val:
            best_val = val_accuracy
            best_svm = svm
            print('best_val : ',best_val)
            print('best_svm : ',best_svm)
```

**Comment by JinwoongKim** [comment]
<img width="609" alt="image" src="https://user-images.githubusercontent.com/12505517/44472791-d891a280-a669-11e8-81e3-f7fb9b17fe66.png">

Previous result in assign 1-2 was 0.39 !!

**Comment by JinwoongKim** [comment]
<img width="676" alt="image" src="https://user-images.githubusercontent.com/12505517/44472851-ffe86f80-a669-11e8-9c8c-0194b84783ca.png">

**Comment by JinwoongKim** [comment]
<img width="256" alt="image" src="https://user-images.githubusercontent.com/12505517/44473534-6a4ddf80-a66b-11e8-9e1b-e39f666d92ac.png">

**Comment by JinwoongKim** [comment]
Validation accuracy:  0.263
lr lrd reg :  0.1 0.3 0.2
Current best accuracy:  0.263
lr lrd reg :  0.1 0.3 0.2
Validation accuracy:  0.266
lr lrd reg :  0.1 0.3 0.25
Current best accuracy:  0.266
lr lrd reg :  0.1 0.3 0.25
Validation accuracy:  0.233
lr lrd reg :  0.1 0.3 0.3
Validation accuracy:  0.254
lr lrd reg :  0.1 0.3 0.35
Validation accuracy:  0.238
lr lrd reg :  0.1 0.3 0.4
Validation accuracy:  0.229
lr lrd reg :  0.1 0.35 0.2
Validation accuracy:  0.228
lr lrd reg :  0.1 0.35 0.25
Validation accuracy:  0.217
lr lrd reg :  0.1 0.35 0.3
Validation accuracy:  0.216
lr lrd reg :  0.1 0.35 0.35
Validation accuracy:  0.206
lr lrd reg :  0.1 0.35 0.4
Validation accuracy:  0.211
lr lrd reg :  0.1 0.4 0.2
Validation accuracy:  0.21
lr lrd reg :  0.1 0.4 0.25
Validation accuracy:  0.202
lr lrd reg :  0.1 0.4 0.3
Validation accuracy:  0.204
lr lrd reg :  0.1 0.4 0.35
Validation accuracy:  0.2
lr lrd reg :  0.1 0.4 0.4
Validation accuracy:  0.202
lr lrd reg :  0.1 0.45 0.2
Validation accuracy:  0.205
lr lrd reg :  0.1 0.45 0.25
Validation accuracy:  0.196
lr lrd reg :  0.1 0.45 0.3
Validation accuracy:  0.192
lr lrd reg :  0.1 0.45 0.35
Validation accuracy:  0.192
lr lrd reg :  0.1 0.45 0.4
Validation accuracy:  0.196
lr lrd reg :  0.1 0.5 0.2
Validation accuracy:  0.196
lr lrd reg :  0.1 0.5 0.25
Validation accuracy:  0.192
lr lrd reg :  0.1 0.5 0.3
Validation accuracy:  0.189
lr lrd reg :  0.1 0.5 0.35
Validation accuracy:  0.161
lr lrd reg :  0.1 0.5 0.4
Validation accuracy:  0.196
lr lrd reg :  0.1 0.55 0.2
Validation accuracy:  0.196
lr lrd reg :  0.1 0.55 0.25
Validation accuracy:  0.189
lr lrd reg :  0.1 0.55 0.3
Validation accuracy:  0.189
lr lrd reg :  0.1 0.55 0.35
Validation accuracy:  0.189
lr lrd reg :  0.1 0.55 0.4
Validation accuracy:  0.199
lr lrd reg :  0.2 0.3 0.2
Validation accuracy:  0.202
lr lrd reg :  0.2 0.3 0.25
Validation accuracy:  0.194
lr lrd reg :  0.2 0.3 0.3
Validation accuracy:  0.193
lr lrd reg :  0.2 0.3 0.35
Validation accuracy:  0.193
lr lrd reg :  0.2 0.3 0.4
Validation accuracy:  0.196
lr lrd reg :  0.2 0.35 0.2
Validation accuracy:  0.203
lr lrd reg :  0.2 0.35 0.25
Validation accuracy:  0.189
lr lrd reg :  0.2 0.35 0.3
Validation accuracy:  0.189
lr lrd reg :  0.2 0.35 0.35
Validation accuracy:  0.193
lr lrd reg :  0.2 0.35 0.4
Validation accuracy:  0.195
lr lrd reg :  0.2 0.4 0.2
Validation accuracy:  0.192
lr lrd reg :  0.2 0.4 0.25
Validation accuracy:  0.188
lr lrd reg :  0.2 0.4 0.3
Validation accuracy:  0.188
lr lrd reg :  0.2 0.4 0.35
Validation accuracy:  0.148
lr lrd reg :  0.2 0.4 0.4
Validation accuracy:  0.193
lr lrd reg :  0.2 0.45 0.2
Validation accuracy:  0.195
lr lrd reg :  0.2 0.45 0.25
Validation accuracy:  0.188
lr lrd reg :  0.2 0.45 0.3
Validation accuracy:  0.202
lr lrd reg :  0.2 0.45 0.35
Validation accuracy:  0.071
lr lrd reg :  0.2 0.45 0.4
Validation accuracy:  0.192
lr lrd reg :  0.2 0.5 0.2
Validation accuracy:  0.196
lr lrd reg :  0.2 0.5 0.25
Validation accuracy:  0.187
lr lrd reg :  0.2 0.5 0.3
Validation accuracy:  0.183
lr lrd reg :  0.2 0.5 0.35
Validation accuracy:  0.128
lr lrd reg :  0.2 0.5 0.4
Validation accuracy:  0.192
lr lrd reg :  0.2 0.55 0.2
Validation accuracy:  0.195
lr lrd reg :  0.2 0.55 0.25
Validation accuracy:  0.187
lr lrd reg :  0.2 0.55 0.3
Validation accuracy:  0.137
lr lrd reg :  0.2 0.55 0.35
Validation accuracy:  0.112
lr lrd reg :  0.2 0.55 0.4
Validation accuracy:  0.194
lr lrd reg :  0.3 0.3 0.2
Validation accuracy:  0.195
lr lrd reg :  0.3 0.3 0.25
Validation accuracy:  0.189
lr lrd reg :  0.3 0.3 0.3
Validation accuracy:  0.188
lr lrd reg :  0.3 0.3 0.35
Validation accuracy:  0.194
lr lrd reg :  0.3 0.3 0.4
Validation accuracy:  0.193
lr lrd reg :  0.3 0.35 0.2
Validation accuracy:  0.195
lr lrd reg :  0.3 0.35 0.25
Validation accuracy:  0.188
lr lrd reg :  0.3 0.35 0.3
Validation accuracy:  0.204
lr lrd reg :  0.3 0.35 0.35
Validation accuracy:  0.082
lr lrd reg :  0.3 0.35 0.4
Validation accuracy:  0.192
lr lrd reg :  0.3 0.4 0.2
Validation accuracy:  0.195
lr lrd reg :  0.3 0.4 0.25
Validation accuracy:  0.186
lr lrd reg :  0.3 0.4 0.3
Validation accuracy:  0.182
lr lrd reg :  0.3 0.4 0.35
Validation accuracy:  0.167
lr lrd reg :  0.3 0.4 0.4
Validation accuracy:  0.194
lr lrd reg :  0.3 0.45 0.2
Validation accuracy:  0.193
lr lrd reg :  0.3 0.45 0.25
Validation accuracy:  0.188
lr lrd reg :  0.3 0.45 0.3
Validation accuracy:  0.215
lr lrd reg :  0.3 0.45 0.35
Validation accuracy:  0.117
lr lrd reg :  0.3 0.45 0.4
Validation accuracy:  0.192
lr lrd reg :  0.3 0.5 0.2
Validation accuracy:  0.195
lr lrd reg :  0.3 0.5 0.25
Validation accuracy:  0.165
lr lrd reg :  0.3 0.5 0.3
Validation accuracy:  0.139
lr lrd reg :  0.3 0.5 0.35
Validation accuracy:  0.151
lr lrd reg :  0.3 0.5 0.4
Validation accuracy:  0.193
lr lrd reg :  0.3 0.55 0.2
Validation accuracy:  0.196
lr lrd reg :  0.3 0.55 0.25
Validation accuracy:  0.19
lr lrd reg :  0.3 0.55 0.3
Validation accuracy:  0.182
lr lrd reg :  0.3 0.55 0.35
Validation accuracy:  0.198
lr lrd reg :  0.3 0.55 0.4
best_val, set :  0.266 (0.1, 0.3, 0.25)
